client/state/data-layer/wpcom/sites/jitm/index.js

```diff
@@ -0,0 +1,166 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { isJetpackSite } from 'state/sites/selectors';
+import { JITM_SET, SECTION_SET, SELECTED_SITE_SET } from 'state/action-types';
+import { makeParser } from 'state/data-layer/wpcom-http/utils';
+import schema from './schema.json';
+
+/**
+ * Poor man's process manager
+ * @type {{
+ * hasInitializedSites: boolean,
+ * hasInitializedSection: boolean,
+ * lastSection: null|string,
+ * lastSite: null|string,
+ * }}
+ */
+const process = {
+	hasInitializedSites: false,
+	hasInitializedSection: false,
+	lastSection: null,
+	lastSite: null,
+};
+
+const unescapeDecimalEntities = str => {
```

Comment:
can we note here in a comment why we only need this and aren't using a normal entity escaper? I don't think it will be obvious to someone coming in later just from the code itself.

Reply:
This is a good point, I need to do another pr after this one to make sure dismissals are synced with wp-admin, so I'll update it there.